pkg/apis/kops/validation/validation.go

```diff
@@ -459,9 +460,30 @@ func validateNetworkingCanal(v *kops.CanalNetworkingSpec, fldPath *field.Path) f
 	return allErrs
 }
 
-func validateNetworkingCilium(c *kops.ClusterSpec, v *kops.CiliumNetworkingSpec, fldPath *field.Path) field.ErrorList {
+func validateNetworkingCilium(cluster *kops.Cluster, v *kops.CiliumNetworkingSpec, fldPath *field.Path) field.ErrorList {
+	c := &cluster.Spec
 	allErrs := field.ErrorList{}
 
+	if v.Version != "" {
+		versionFld := fldPath.Child("version")
+		version, err := semver.ParseTolerant(v.Version)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(versionFld, v.Version, "Could not parse as semantic version"))
+		}
+
+		v8, _ := semver.Parse("1.8.0")
+		v7, _ := semver.Parse("1.7.0")
+		v6, _ := semver.Parse("1.6.0")
+
+		if !(version.GTE(v6) && version.LT(v8)) {
+			allErrs = append(allErrs, field.Invalid(versionFld, v.Version, "Only versions 1.6 and 1.7 are supported"))
+		}
+
+		if !cluster.IsKubernetesGTE("1.12.0") && version.GTE(v7) {
```

Comment:
This would permit "1.7.0-rc1" on Kubernetes 1.11. You might want to use `util.IsKubernetesGTE()`

Reply:
Just ignoring the pre/build parts similar to what `util.IsKubernetesGTE()` does.

Reply:
The usual pattern is to use just `"1.12"`